timelock-agent/src/main/java/com/palantir/timelock/paxos/AsyncTimeLockServicesCreator.java

```diff
@@ -102,6 +102,7 @@ private static AsyncTimelockService createRawAsyncTimelockService(
     }
 
     private static <T> T instrument(Class<T> serviceClass, T service, String client) {
-        return AtlasDbMetrics.instrument(serviceClass, service, MetricRegistry.name(serviceClass, client));
+        // TODO(nziebart) tag with the client name, when tritium supports it
```

Comment:
working out spec and implementation this week/next, initial strawman in https://github.com/palantir/tritium/pull/39

More broadly, we may want to reassess all Atlas metrics after we have tag support as well